discord/utils.py

```diff
@@ -1374,3 +1374,8 @@ def _shorten(
 
 def _to_kebab_case(text: str) -> str:
     return CAMEL_CASE_REGEX.sub('-', text).lower()
+
+
+def strip_namedtuple_docs(cls: NamedTuple) -> None:
```

Comment:
This also shouldn't pass type checking I don't think? I'd really advise running Pyright on this PR

Reply:
I've switched to using dataclasses so this is irrelevant now.